cfpb/consumerfinance.gov
# Form explainers: remove tab references from tracking code

We are about to migrate from the hard-coded Loan Estimate and Closing Disclosure explainers to new Wagtail versions. The new pages don't use tabs, so we need to remove the tab references from the explainers gtm code.

## Changes

- Remove tab references from explainers tracking code


## How to test this PR

1. Check out branch and run `yarn scripts`
2. With a recent db dump, find and publish the draft Closing Disclosure `Form explainer` page
3. Enable the `WAGTAIL_FORM_EXPLAINERS` flag
4. Navigate to [Closing disclosure](http://localhost:8000/owning-a-home/closing-disclosure/) page and verify that there are no gtm console errors when interacting with the expandables

## Notes and todos

- We'll check with Analytics to see if this tracking code is still necessary and, if not, remove it along with the rest of the hard-coded explainer code in a future PR. 


## Checklist


  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
